spec/tree_spec.rb

```diff
@@ -1,14 +1,137 @@
 require 'rspec'
 require 'tree'
 
+def kill_tree(tree)
+  # The 12 here is based off of the rand_seed the below
+  12.times { tree.age! }
```

Comment:
this works but its preferred to either accomplish this with a helper method or `let`. https://relishapp.com/rspec/rspec-rails/v/3-7/docs/helper-specs/helper-spec